roles/db-start/templates/start_all.sh.j2

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+while IFS=':' read -r ORACLE_SID ORACLE_HOME AUTOSTART_FLAG; do
+    if [[ "$AUTOSTART_FLAG" =~ [Yy] ]]; then
+        PATH_SAVE="$PATH"
+        LD_LIBRARY_PATH_SAVE="$LD_LIBRARY_PATH"
+        export ORACLE_SID ORACLE_HOME
+        export PATH="$ORACLE_HOME/bin:$PATH"
+        export LD_LIBRARY_PATH="$ORACLE_HOME/lib:$LD_LIBRARY_PATH"
+        if ! pgrep -f "tnslsnr.*{{ listener_name }}" ; then
+            echo "Starting listener {{ listener_name }}"
+            "$ORACLE_HOME"/bin/lsnrctl start {{ listener_name }}
+        fi
+        if ! pgrep -f "ora_smon_$ORACLE_SID" ; then
+            # startup is in separate sqlplus calls due to dbms_output not working correctly
+            echo "Starting Oracle instance $ORACLE_SID"
+            sqlplus -S / as sysdba <<EOF
+WHENEVER OSERROR EXIT FAILURE
+WHENEVER SQLERROR EXIT FAILURE
+STARTUP MOUNT;
+EXIT;
+EOF
+            if [ $? -ne 0 ]; then
+                echo "ERROR: Failed to mount $ORACLE_SID"
```

Comment:
If we hit a fatal error like this and can't open the database, I think we'd want to return an errror code to the caller, so that systemd can inform the user that it failed.  (I think that, otherwise, systemd just hides the output of the script).  But I also agree that we should continue the loop, and attempt to start all the databases, before returning the error to the OS.

And as per my other comment here, we probably want to address the "database already open" case so that it doesn't report errors.